explainers/doc/CREATING_CUSTOM_EXPLAINER_WITH_MLI_BYOR.md

```diff
@@ -42,7 +42,7 @@ When MLI user starts interpretation, model compatible explainers (from the avail
 ### What is the role of recipes?
```

Comment:
Is all of this suppose to go into the DAI user guide? If so, then it should be `*.rst` files. Right @zainhaq-h2o @NidhiMehta ?

Reply:
@navdeep-G I can definitely add this info to the DAI user guide if the MLI team thinks it's a good idea to have it included there.

Reply:
@navdeep-G yes, I think that it's desired to make this document part of DAI documentation.

@zainhaq-h2o @NidhiMehta will you convert it to `*.rst` or is it expected that I will do it myself. Please let me know.

Reply:
@dvorka I will convert it for you. I will create a PR and add you as a reviewer.

Reply:
Thank you Zain - I really appreciate it!